v3.0/reference/sql/character-set.md

```diff
@@ -254,4 +254,10 @@ SET collation_connection = @@collation_database;
 * 规则2:    指定 CHARACTER SET charset_name 且未指定 COLLATE collation_name，则使用 CHARACTER SET charset_name 和 CHARACTER SET charset_name 默认的排序比较规则。
 * 规则3:   CHARACTER SET charset_name 和 COLLATE collation_name 都未指定，则使用更高优化级给出的字符集和排序比较规则。
 
+## 字符合法性检查
+
+当指定的字符集为 utf8 或 utf8mb4 时，TiDB 仅支持合法的 utf8 字符。对于不合法的字符，会报错：`incorrect utf8 value`。
```

Comment:
Please add the following details like what you mentioned in description:

```
MySQL 5.7 has no check for character but MySQL 8.0 has. This could make confusion. So I
declared the action of TiDB.
```

Reply:
addressed.